docs/using-nativeaot/reflection-in-aot-mode.md

```diff
@@ -69,10 +69,27 @@ In CoreRT, reflection metadata (names of types, list of their methods, fields, s
 
 If compiler cannot detect types used by the aplication, an rd.xml file can be supplemented to help ILCompiler find types that should be analyzed.
 For that, file `rd.xml` should be created and following lines added to project file
-```
+```xml
 <ItemGroup>
   <RdXmlFile Include="rd.xml" />
 </ItemGroup>
 ```
 
 Format of the file described [here](rd-xml-format.md)
+
+## Shimming
+
```

Comment:
We have a few more of these shim settings. They can be documented in separate PRs.